### DISCUSSION TITLE
Is it possible to add safari support?

### DISCUSSION THREAD

**Discussion by jimmymcheung** [ClearURLs/Addon#283] [Ideas]
I quickly have a glance on the language this project has used, which appears be mainly JS and CSS, I think it is pretty cross browser compatible. Is it possible to add packaging script so user without much experience could compile it to a safari extension? I suggest this because of cause the cost of an apple developer account is not little, and one might find it annoying/unsafe to trust before enabling. Let user compile it might be a walk around, but correct me if that's not true.

**Comment by inson1** [comment]
duplicate of #92
